extra/platform.xsd

```diff
@@ -35,6 +35,12 @@
             </xs:extension>
         </xs:simpleContent>
     </xs:complexType>
+    <xs:simpleType name="noiseType">
```

Comment:
this should be referenced in line 30:
`<xs:attribute type="xs:string" name="type"/>` --> `<xs:attribute type="noiseType" name="type"/>`